main.go

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 // Package main implements HarbourBridge, a stand-alone tool for Cloud Spanner
-// evaluation, using data from an existing PostgreSQL database. See README.md
+// evaluation, using data from an existing PostgreSQL/MYSQL database. See README.md
```

Comment:
MYSQL -> MySQL

Reply:
Done. Also checked other places where it would be Mysql or MYSQL instead of MySQL.